cargo-dist/src/tasks.rs

```diff
@@ -3066,3 +3084,36 @@ fn submodule_head(submodule_path: &Utf8PathBuf) -> DistResult<Option<String>> {
         Ok(Some(commit.to_owned()))
     }
 }
+
+// Picks the newest glibc in the case of conflicts
+fn glibc_thunderdome(glibcs: Vec<GlibcVersion>) -> GlibcVersion {
```

Comment:
10/10 name

(this is a normal comment when it can/should be a doc-comment for intellisense)